devices/src/virtio/block.rs

```diff
@@ -278,7 +278,7 @@ impl Request {
     }
 }
 
-struct BlockEpollHandler {
+pub struct BlockEpollHandler {
```

Comment:
Please add documentation for everything `pub`. Add `#![deny(missing_docs)]` to `virtio/mod.rs` to make sure everything public is documented.

Reply:
I added documentation for the structs and methods that I made public as part of this PR. Adding documentation for everything else in this crate would require significantly more effort. I wouldn't do it as part of this PR. I don't know if it's worth doing at all since we don't plan on publishing this crate in rust-vmm. But if it is, I propose creating a different issue for this.